### PR TITLE
chore: fix Prisma Client naming inconsistencies ("the Client")

### LINKED ISSUE #3192: [Docs] Fix inconsistent references to Prisma Client
In the docs, we refer to generating "the client" or "Prisma Client" inconsistently. Unless there's some distinction here that I'm missing, I think we should make these mentions consistent - pick one name and go with it. I'd suggest "Prisma Client". Can we fix this en masse?

### PR DESCRIPTION
**Pull request by ruheni**
Closes https://github.com/prisma/docs/issues/3192

### CODE REVIEW

**Review by nikolasburk — APPROVED** [review]
very nice!

File: content/200-concepts/100-components/01-prisma-schema/08-views.mdx [hunk 1]
```diff
@@ -179,7 +179,7 @@ To define the `UserInfo` view from the example above, begin by using the `view`
 
 ```prisma
 view UserInfo {
-  // Fields
+// Fields
```
Comment (janpio) [inline content/200-concepts/100-components/01-prisma-schema/08-views.mdx:182]: broken indentation

File: content/200-concepts/100-components/01-prisma-schema/08-views.mdx [hunk 2]
```diff
@@ -199,10 +199,10 @@ The fields of the `UserInfo` example view can be defined as follows:
...
 ```prisma highlight=2-5;normal
 view UserInfo {
-  id    Int    @unique
-  email String
-  name  String
-  bio   String
+id    Int    @unique
+email String
+name  String
+bio   String
```
Comment (janpio) [inline content/200-concepts/100-components/01-prisma-schema/08-views.mdx:205]: broken indentation

File: content/200-concepts/100-components/01-prisma-schema/08-views.mdx [hunk 3]
```diff
@@ -211,10 +211,10 @@ view UserInfo {
...
 ```prisma highlight=2-5;normal
 view UserInfo {
-  id    String  @id @default(auto()) @map("_id") @db.ObjectId
-  email String
-  name  String
-  bio   String
+id    String  @id @default(auto()) @map("_id") @db.ObjectId
+email String
+name  String
+bio   String
```
Comment (janpio) [inline content/200-concepts/100-components/01-prisma-schema/08-views.mdx:217]: broken indentation

File: content/200-concepts/100-components/01-prisma-schema/08-views.mdx [hunk 4]
```diff
@@ -240,14 +240,13 @@ npx prisma db pull
 The resulting `view` block will be defined as follows:
 
 ```prisma
-/// The underlying view does not contain a valid unique identifier and can therefore currently not be handled by the Prisma Client.
+/// The underlying view does not contain a valid unique identifier and can therefore currently not be handled by Prisma Client.
```
Comment (janpio) [inline content/200-concepts/100-components/01-prisma-schema/08-views.mdx:243]: This quotes an error message - if the error message is grammatically incorrect, this has to be fixed in the actual error message code, not just where we quote it.

File: content/200-concepts/100-components/01-prisma-schema/08-views.mdx [hunk 5]
```diff
@@ -240,14 +240,13 @@ npx prisma db pull
...
-  email String?
-  name  String?
-  bio   String?
-
-  @@ignore
+id    Int?
+email String?
+name  String?
+bio   String?
+@@ignore
```
Comment (janpio) [inline content/200-concepts/100-components/01-prisma-schema/08-views.mdx:249]: broken indentation

File: content/200-concepts/100-components/01-prisma-schema/08-views.mdx [hunk 6]
```diff
@@ -273,15 +272,14 @@ In order to make this `view` block valid you will need to:
 - Remove the comment Prisma generated warning about an invalid view
 
 ```prisma highlight=4;add|1,3,8,9;delete
-/// The underlying view does not contain a valid unique identifier and can therefore currently not be handled by the Prisma Client.
+/// The underlying view does not contain a valid unique identifier and can therefore currently not be handled by Prisma Client.
```
Comment (janpio) [inline content/200-concepts/100-components/01-prisma-schema/08-views.mdx:275]: see above

File: content/200-concepts/100-components/01-prisma-schema/08-views.mdx [hunk 7]
```diff
@@ -273,15 +272,14 @@ In order to make this `view` block valid you will need to:
...
-  name  String?
-  bio   String?
-  
-  @@ignore
+id    Int?
+id    Int    @unique
+email String?
+name  String?
+bio   String?
+@@ignore
```
Comment (janpio) [inline content/200-concepts/100-components/01-prisma-schema/08-views.mdx:282]: broken indentation

Comment (ruheni) [reply content/200-concepts/100-components/01-prisma-schema/08-views.mdx:282]: Resolved in this PR: https://github.com/prisma/docs/pull/4690

File: content/200-concepts/100-components/01-prisma-schema/08-views.mdx (hunk 5 above)
Comment (ruheni) [reply content/200-concepts/100-components/01-prisma-schema/08-views.mdx:249]: Resolved here: https://github.com/prisma/docs/pull/4690

**Pull request merged**